.github/workflows/init_repository.yml

```diff
@@ -0,0 +1,110 @@
+#
+# Copyright 2022 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Stage a new root and targets
+
+permissions: read-all
+
+# Execute this ONLY as workflow dispatch.
+on:
+  workflow_dispatch:
+    inputs:
+      revoke_key:
+        description: 'If provided, revokes the given HSM key, identified by the serial number'
+        required: false
+        type: string
+      prev_repo:
+        description: 'Chains the new root from this previous repository'
+        required: false
+        default: `repository/`
+        type: string
+
+jobs:
+  init_root:
+    runs-on: ubuntu-20.04
+    permissions:
+      id-token: 'write'
+    steps:
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # v2.4.0
+        with:
+          fetch-depth: 0
+      # TODO(https://github.com/sigstore/root-signing/issues/98): Use a common configuration checked into source control
+      - name: setup environment
+        run: |
+          echo "GITHUB_USER=${{ github.actor }}" >> $GITHUB_ENV
+          echo "SNAPSHOT_KEY=gcpkms://projects/sigstore-root-signing/locations/global/keyRings/root/cryptoKeys/snapshot" >> $GITHUB_ENV
```

Comment:
To confirm, have these keys been created and does the GitHub Actions workflow have access to them?

Reply:
Yep